internal/firewall/iptables.go

```diff
@@ -179,6 +179,17 @@ func (c *Config) acceptOutputFromIPToSubnet(ctx context.Context,
 	return c.runIP6tablesInstruction(ctx, instruction)
 }
 
+func (c *Config) acceptIpv6MulticastOutput(ctx context.Context,
+	intf string, remove bool) error {
+	interfaceFlag := "-o " + intf
+	if intf == "*" { // all interfaces
+		interfaceFlag = ""
+	}
+	instruction := fmt.Sprintf("%s OUTPUT %s -d ff00::/8 -j ACCEPT",
```

Comment:
Forgive my eventual IPv6 ignorance, but isn't `ff00::/8` too permissive? I see this method is used next to `acceptOutputFromIPToSubnet` which is more 'narrow' in terms of permission